src/main/java/teammates/common/datatransfer/attributes/CourseAttributes.java

```diff
@@ -106,10 +106,6 @@ public String getDeletedAtFullDateTimeString() {
         return TimeHelper.formatDateTimeForDisplay(localDateTime);
     }
 
-    public void setDeletedAt() {
```

Comment:
How about `resetDeletedTime`?

If it is only used in the test code, I think we can remove it.